msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,348 @@
+> [!TIP]
+> Some guidelines may not be applicable to your app. For example, if your app doesn't include a bot, you can ignore bot-related guidelines.
+
+## 1.0 Value proposition
+
+### 1.1 App name
+
+An app's name plays a critical role in how users discover it in the store. Remember the following about app names:
+
+* The name must include terms relevant to your users.
+* It must not copy the name of an existing app in the store or other offer in the commercial marketplace.
+* Common nouns must be prefixed or suffixed with the publisher’s name (for example, “XYZ Tasks” rather than “Tasks”)
+* It must not use **Teams** or other Microsoft product names that could falsely indicate co-branding or co-selling. (For more information about referencing Microsoft software, products, and services, see the [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general)).
+* If your app is part of an official partnership with Microsoft, the name of your app must come first (for example, Salesforce Connector for Microsoft Teams).
+
+### 1.2 Suitable for workplace consumption
+
+App content must be suitable for general workplace consumption and abide by all restrictions listed in the [commercial marketplace certification policies](https://docs.microsoft.com/legal/marketplace/certification-policies#10010-inappropriate-content). Content related to religion, politics, gambling, and prolonged entertainment is prohibited.
+
+Your app must facilitate group collaboration, improve an individual's productivity, or both. Apps intended for team bonding and socializing must be collaborative and designed for multiple participants. These types of apps also should not require a substantial time investment or perceptively impact productivity.
+
+### 1.3 Similar platforms and services
+
+Apps must focus on the Teams experience and not include the names, icons, or imagery of other similar chat-based collaboration platforms or services unless the apps provide specific interoperability.
+
+## 2.0 Security
+
+### 2.1 Publisher Attestation
+
+The Microsoft 365 App Compliance Program is intended to help organizations assess and manage risk by evaluating security and compliance information about your app.
+
+Teams app developers must complete the Publisher Attestation portion of the program. For more information, see the [Microsoft 365 App Compliance Program documentation](https://docs.microsoft.com/microsoft-365-app-certification/docs/attestation).
+
+### 2.2 Financial information
+
+Your app must not ask users to make payments within the Teams interface. Financial instrument details must not be transmitted through the user to the bot interface. Links to secure payment services may only be transmitted to users if disclosure has been made in the app’s terms of use, privacy policy, or any profile page or website before the user agrees to use the bot.
```

Comment:
@heath-hamilton, Please update the section to  -
 
Your app must not ask users to make payments within the Teams interface. Financial instrument details must not be transmitted to the user through the bot interface. 
Links to secure payment services may only be transmitted to users if disclosure has been made in the app’s terms of use or privacy policy before the user agrees to use the application.

iOS and Android versions of apps should follow – 
•	App must not include any in-app purchases, trial offers, UI that aims to upsell to paid versions, or links to any online stores where users can purchase or acquire other content, apps, or add-ins.
•	If the add-in requires an account, accounts may only be created if there is no charge; the use of the term "free" or "free account" is not allowed. 
•	You may determine whether the account is active indefinitely or for a limited time, but if the account expires, no UI, text, or links indicating the need to pay may be shown.
•	The associated Privacy Policy and Terms of Use pages must also be free of any commerce UI or Store links.

Reply:
@ramendhe Is the "iOS and Android versions" speaking to apps running in the *Teams* version of iOS or Android? Also, I think this info is better suited closer to the "Advertising" section and not within "Security".

Reply:
@heath-hamilton, this refers to 3PApps that run in Teams app on iOS & Android.
This info may not be directly related to 'Security', but its linked to financial information & implication to Apple policies for providing any purchase option that is not done via apple account. refer - https://docs.microsoft.com/en-us/legal/marketplace/certification-policies#11202-acquisition-pricing-and-terms

Its not related to advertise. For consistency let's keep it in financial section under security